Лабораторная 2

BoAnDm opened the pull request:
![image](https://github.com/user-attachments/assets/3d57b37d-e627-48d7-bbaa-1564c23eed09)

BoAnDm committed:
Лабораторная 2

meat9 commented on task_2.py:
```diff
@@ -0,0 +1,57 @@
+        for index, book in enumerate(self.books):
+            if id_ == book.id_:
+                return index
+        print("Книги с запрашиваемым id не существует")
```
необходимо вернуть ошибку с типом value error а не вывести сообщение через принт

BoAnDm replied:
Исправила

BoAnDm committed:
Update task_2.py

Исправила (необходимо вернуть ошибку с типом value error а не вывести сообщение через принт):

for index, book in enumerate(self.books):
            if id_ == book.id_:
                return index
        print("Книги с запрашиваемым id не существует")

на:
raise ValueError("Книги с запрашиваемым id не существует")
